HealthGPT/HealthGPT/MessageInputView.swift

```diff
@@ -35,149 +25,13 @@ struct MessageInputView: View {
                 .background(Color(.systemGray6))
                 .cornerRadius(10)
                 .lineLimit(1...5)
-                .disabled(isQuerying == true)
+                .disabled(messageManager.isQuerying)
 
             Button(action: {
-                isQuerying = true
```

Comment:
Nice, thank you for cleaning up this long closure here!
We can therefore probably reenable the SwiftLint warnings for long closures in this project.

Reply:
I have removed the comment disabling `closure_body_length`. Thanks!